examples/Communication_example/communication_example_utils.py

```diff
@@ -0,0 +1,82 @@
+import numpy as np
+import pandas as pd
+
+
+def get_known_actor_comms_status(values):
+    """Helper function to track comms status"""
```

Comment:
Inputs shall be described in the docstring. You can check other files for examples.
(e.g., values (float): values are....
Returns: 
      return_name : what it is.

Please do the same for the other functions.